### PR TITLE
Remove CRDs from helm completely

### PR DESCRIPTION
**Pull request by MarcelMue**
This is basically a sledgehammer fix to the issue. It completely removes the CRDs from helm and hardcodes the names which we use in our tests.

As you can see this works, but is not a sustainable solution. I will create a follow up issue to come up with a long term solution before release 1.4.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -628,6 +628,10 @@ jobs:
       - name: install harbor-operator
         run: |
           set -ex
+          make kustomize
+
+          ./bin/kustomize build --reorder legacy config/helm/crds | kubectl create -f -
```
Comment (thcdrt) [inline .github/workflows/tests.yml:633]: Shouldn't we add a Makefile target to install crds ?

File: config/helm/crds/kustomization.yaml [hunk 2]
```diff
@@ -1,12 +1,12 @@
-namePrefix: '{{ include "chart.fullname" . }}-'
+namePrefix: 'harbor-operator-'
 
-namespace: '{{ .Release.Namespace }}'
+namespace: 'harbor-operator-ns'
```
Comment (thcdrt) [inline config/helm/crds/kustomization.yaml:3]: So we can't install CRDs in other namespace than harbor-operator-ns ?

**Review by thcdrt — APPROVED** [review]

File: .github/workflows/tests.yml (hunk 1 above)
Comment (MarcelMue) [reply .github/workflows/tests.yml:633]: This is how it is done in the other tests currently - I am happy to create an issue to align this and not do this crazy stuff anymore :D

**Comment by MarcelMue** [comment]
I created #1041 as a follow up.

**Pull request merged**